packages/mjml-divider/src/index.js

```diff
@@ -16,6 +16,7 @@ export default class MjDivider extends BodyComponent {
     'padding-right': 'unit(px,%)',
     'padding-top': 'unit(px,%)',
     width: 'unit(px,%)',
+    align: 'enum(left, center, right)',
```

Comment:
Here there must be no spaces before 'center' and 'right', otherwise the validator expects those spaces